arm-compute/compositeComputeClient.json

```diff
@@ -4,7 +4,7 @@
         "description": "Composite Swagger for Compute Client"
     },
   "documents": [
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2015-06-15/swagger/compute.json",
+    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/compute.json",
```

Comment:
nope. This is not supported for now. AutoRest does not know that "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/" is the base url and you are referring another document relative to the base.
Anyways, using the full url is easy for now. We will think of adding basePath and then have documents relative to the basepath. But this is of low priority in the current scheme of things.